client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -71,7 +72,9 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep }: Sidebar
 		const { domain, isPrimary, isWPCOMDomain, sslStatus } = sidebarDomain;
 
 		[ siteName, topLevelDomain ] = getUrlInfo( domain );
-		showClipboardButton = isWPCOMDomain ? true : sslStatus === 'active' && isPrimary;
+
+		isDomainSSLProcessing = sslStatus !== 'active' && sslStatus !== null;
```

Comment:
[QUESTION] Out of curiosity (and aware these achieve the same aim) is there a style rule/reason for preferring this approach vs. `sslStatus && sslStatus !== 'active'` or `sslStatus && (sslStatus === 'pending' || sslStatus === 'disabled')`? 

Also, what are the general thoughts around using enums to maintain string values?

Reply:
> Out of curiosity (and aware these achieve the same aim) is there a style rule/reason for preferring this approach vs. sslStatus && sslStatus !== 'active' or sslStatus && (sslStatus === 'pending' || sslStatus === 'disabled')?

Definitely no reason for preferring one approach over the other. I didn't give the conditional too much thought and spent more time figuring out the right name for the `isDomainSSLProcessing` variable ( to capture the intention of what I was doing ).

If anything, either of your two suggestions seems much clearer and I'm happy to swap the logic

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/69317/commits/3cdf0a66d73777a4bf9d7e9b19540d2e5c8cd966